lib/install/bin/webpack-dev-server.tt

```diff
@@ -10,25 +10,30 @@ RAILS_ENV = ENV["RAILS_ENV"]
 ENV["NODE_ENV"] ||= RAILS_ENV
 NODE_ENV = ENV["NODE_ENV"]
 
-APP_PATH = File.expand_path("../", __dir__)
+APP_PATH          = File.expand_path("../", __dir__)
+CONFIG_FILE       = File.join(APP_PATH, "config/webpacker.yml")
+NODE_MODULES_PATH = File.join(APP_PATH, "node_modules")
+
+begin
+  yaml_config = YAML.load_file(CONFIG_FILE)[NODE_ENV]
+  paths       = yaml_config["paths"]
+  dev_server  = yaml_config["devServer"]
+
+  WEBPACK_CONFIG  = File.join(APP_PATH, paths["config"], "development.js")
+  DEV_SERVER_HOST = "http#{"s" if dev_server["https"]}://#{dev_server["host"]}:#{dev_server["port"]}"
```

Comment:
Instead of dropping `ARGV` entirely, maybe we should parse it and use the relevant options when supplied. Something along the lines of `#{options["host"] || dev_server["host"]}`. WDYT?